newrelic/core/application.py

```diff
@@ -1238,6 +1245,28 @@ def harvest(self, shutdown=False, flexible=False):
                                 data_sampler.name,
                             )
 
+                    # Send environment plugin list
+
+                    stopwatch_start = time.time()
+                    while (
+                        configuration
+                        and configuration.package_reporting.enabled
+                        and self._env_send
+                        and ((time.time() - stopwatch_start) < 2.0)
+                    ):
+                        try:
+                            module_info = next(self.plugins)
+                            self.modules.append(module_info)
+                        except StopIteration:
+                            self._env_send = False
```

Comment:
Can we name this something more explicit to what it's actually used for? Instead of `_env_send` perhaps `_plugins_to_iterate` or `_remaining_plugins`.